src/main/scala/com/example/PositionOfShips.scala

```diff
@@ -28,12 +29,16 @@ class PositionOfShips(positions: Vector[Position]) {
     val totalDistance =
       pos
         .sliding(2)
-        .map { case Vector(a, b) =>
+        .map {
+        case Vector(a) => 0.0
```

Comment:
Kannst Du diese Änderungen in einen extra Commit rebases? Dann würde ich denn schon pullen. (Und sehen, ob das überhaupt so geht, wie ich mir das vorstelle mit Git)